src/main/java/com/dev/cinema/Main.java

```diff
@@ -39,5 +42,18 @@ public static void main(String[] args) {
         movieSessionService.add(movieSession);
         System.out.println(movieSessionService
                 .findAvailableSessions(movie.getId(), LocalDate.now()));
+
+        User user = new User();
+        user.setEmail("bob");
+        user.setPassword("123");
```

Comment:
there is no need to do this here - User is created in and returned from `register()` method